okhttp/src/test/java/okhttp3/CallTest.kt

```diff
@@ -2855,7 +2869,7 @@ open class CallTest {
     server.enqueue(MockResponse())
     executeSynchronously("/")
     val recordedRequest = server.takeRequest()
-    assertThat(recordedRequest.headers["User-Agent"]).matches(userAgent)
+    assertThat(recordedRequest.headers["User-Agent"]!!).isEqualTo(userAgent)
```

Comment:
Shouldn't this be supported without !!

https://github.com/willowtreeapps/assertk/blob/5ff8a09f63543a18365dee1bb48dc9c4684aab9e/assertk/src/commonMain/kotlin/assertk/assertions/string.kt#L6